host/src/containers/GameSessionContainer.tsx

```diff
@@ -104,20 +112,66 @@ const GameSessionContainer = () => {
     }
   }, [isModalOpen]);
 
+  useEffect(() => { //headerGame timer, 1 second interval, update localstorage in case page resets
```

Comment:
nit: The common practice is:
- Having the comments to be in a new line
- A space after `//`

Reply:
went through `GameSessionContainer`, `StartGame`, `GameInProgress` and applied new line + space after // to longer comments